promptsource/templates/tau/scrolls/gov_report/templates.yaml

```diff
@@ -0,0 +1,87 @@
+dataset: tau/scrolls
+subset: gov_report
+templates:
+  2edf0297-e3d4-4870-bd95-178411e52e09: !Template
+    answer_choices: null
+    id: 2edf0297-e3d4-4870-bd95-178411e52e09
+    jinja: 'Summary: {{ output }}
+    
+    Full Document: ||| {{ input }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      languages:
+      - en
+      metrics:
+      - ROUGE
+      - BLEU
+      original_task: false
+    name: summary_doc
+    reference: ''
+  3bd082cb-4e28-4eb7-1ea2-dd03f1f86219: !Template
+    answer_choices: null
+    id: 3bd082cb-4e28-4eb7-1ea2-dd03f1f86219
+    jinja: 'Read the following document:
+
+    {{ output }}
+
+    The following is a concise summary of the document: ||| {{ input }}'
```

Comment:
I'm still weirded out that you can have trailing "\n"

Reply:
Sorry where's the trailing "\n"? Maybe I'm missing something

Reply:
sorry *can't I would expect the generation to be "below" the "The following is a concise summary of the document:" instead of to the right.

Reply:
Oh I see. Yeah does promptsource allow for that? Not too familiar with the convention in PS / why it can't have a trailing "\n"

Reply:
It doesn't allow for it; \n & whitespace is stripped